cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -241,8 +248,16 @@ function move_sriov_pfs_netns_to_node {
 
     ip link set "$pf_name" netns "$node"
     pf_array+=("$pf_name")
+    if [ "${#pf_array[@]}" -eq "$num_pfs_required" ]; then
+      break
+    fi
   done
 
+  if [ "${#pf_array[@]}" -lt "$num_pfs_required" ]; then
+    echo "FATAL: Could not allocate enough PFs, please check PF_BLACKLIST, PFS_COUNT_PER_NODE, and how many PF actually available or used already"
```

Comment:
Please provide the values that caused the error so the user can see what happened.